### PR TITLE
update html

### PR DESCRIPTION
**Pull request by sc369**

### CODE REVIEW

**Review by prafullatass — APPROVED** [review]

File: personal.html [hunk 1]
```diff
@@ -11,82 +11,55 @@
...
-             <section class ="familyMembers" id="twins"></section>
-                <img id ="twins_pic" src="https://www.essence.com/wp-content/uploads/2018/07/1532613703/beyonce%20twins-445x297.jpeg" alt="twins">           
-                          
-            On June 13, 2017, Beyonce gave birth to twins, Rumi and Sir Carter. The twins were delivered by Dr Paul Crane at Ronald Reagan UCLA Medical Center in Los Angeles. 
     </article>
 
-
+    </div>
 </body>
+<footer>&copy; Copyright 2019 HTML.am</footer>
```
Comment (AjeffersonTN) [inline personal.html]: Please remove the "&copy; Copyright 2019 HTML.am" on line 60 and add the closing footere tag at the end of line 61.

**Review by AjeffersonTN — CHANGES_REQUESTED** [review]
See line 60 comments

**Comment by sc369** [comment]
Copyright updated

**Review by AjeffersonTN — APPROVED** [review]
👍

**Pull request merged**